dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/GroupRestRepository.java

```diff
@@ -33,11 +40,45 @@
 
 @Component(GroupRest.CATEGORY + "." + GroupRest.NAME)
 public class GroupRestRepository extends DSpaceRestRepository<GroupRest, UUID> {
-    GroupService gs = EPersonServiceFactory.getInstance().getGroupService();
+    @Autowired
+    GroupService gs;
 
     @Autowired
     GroupConverter converter;
 
+    @Override
+    protected GroupRest createAndReturn(Context context)
```

Comment:
This method/endpoint will likely be accessible to anonymous users as-is. I think we need to access restrict this with an @PreAuthorize annotation. Likely should be:
`@PreAuthorize("hasAuthority('ADMIN')")`

For more details on this annotation, see: https://wiki.duraspace.org/display/DSPACE/REST+Authorization